src/social/pages/CommunityPost/index.js

```diff
@@ -1,24 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
 
+import { PrimaryButton, ButtonGroup } from '~/core/components/Button';
+import useUser from '~/core/hooks/useUser';
+import { isCoach } from '~/helpers/permissions';
 import withSDK from '~/core/hocs/withSDK';
 import CommunityInfo from '~/social/components/CommunityInfo';
 import Post from '~/social/components/post/Post';
+import { useConfig } from '~/social/providers/ConfigProvider';
 
 import { Wrapper } from './styles';
 
 const CommunityPost = ({
+  currentUserId,
   postId,
   commentId,
   communityId,
   readonly,
   handleCopyPostPath,
   handleCopyCommentPath,
 }) => {
+  const { fetchNextPostInCommunity } = useConfig();
+  const { user: currentUser } = useUser(currentUserId, [currentUserId]);
+  const navigate = useNavigate();
+
+  const onOlderPost = async (communityId, postId) => {
+    const nextPostPath = await fetchNextPostInCommunity(communityId, postId, "AFTER");
+    navigate(nextPostPath);
+  };
+  const onNewerPost = async () => {
+    const nextPostPath = await fetchNextPostInCommunity(communityId, postId, "BEFORE");
+    navigate(nextPostPath);
+  };
+
   return (
     <Wrapper>
       <CommunityInfo communityId={communityId} />
 
+      {
+      isCoach(currentUser) &&
```

Comment:
Nit: Add some margin to the bottom and adjust ButtonGroup spacing so it matches the spacing with the top card.